.jenkins/Jenkinsfile

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/groovy
+@Library('jenkins-pipeline@v0.4.4')
+import com.invoca.utils.*;
+
+APPRAISAL_JOB_NAME = '/Aggregate Gem/Appraisal Test'
+
+pipeline {
+  agent {
+    kubernetes {
+      defaultContainer 'ruby'
+      yamlFile '.jenkins/ruby_build_pod.yml'
+    }
+  }
+
+  environment { GITHUB_TOKEN = credentials('github_token') }
+
+  stages {
+    stage('Setup') {
+      steps {
+        updateGitHubStatus('clean-build', 'pending', "Running unit tests")
+        sh 'bundle install'
+        sh 'bundle exec rake db:migrate'
+      }
+    }
+
+    stage("Tests") {
+      parallel {
+        stage('Unit Tests') {
+          steps { sh "bundle exec rake test" }
+          post  { always { junit 'test/reports/*.xml' } }
+        }
+
+        stage("Rails 4 Appraisal") {
+          steps {
+            build(
```

Comment:
So these wind up running in their own workspace?

Reply:
yea, each clone and job that runs gets it's own workspace 😄